server/routes/api/lob.js

```diff
@@ -8,22 +8,20 @@ router.get('/:template_id', async (req, res) => {
     const template_id = req.params.template_id;
     var templateinfo = {};
 
-    try {
-        axios
```

Comment:
The original problem here was a missing `await` keyword here. I've essentially reverted this PR and added the `await` keyword in #79.